package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "reactifex",
-  "version": "1.1.1",
+  "name": "@edx/reactifex",
+  "version": "0.0.0-development",
```

Comment:
In other libraries, we set this version to `"1.0.0-semantically-released"` - I don't know exactly why we do that, to be clear, but being consistent seems worthwhile unless we have a good reason not to.

Example here: https://github.com/edx/frontend-build/blob/master/package.json#L3